Packs/Base/Scripts/CommonServerPython/CommonServerPython.py

```diff
@@ -36,9 +37,9 @@ def __line__():
     return cf.f_back.f_lineno
 
 
-# 41 - The line offset from the beggining of the file.
+# 42 - The line offset from the beggining of the file.
```

Comment:
Did you remember to fix it, or did the test fail?

Reply:
test fail